### PR TITLE
Migrated latest docs.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Vedant Shrotria <vedant.shrotria@mayadata.io>

**What this PR does / why we need it**:

**This PR will migrate the latest docs from litmus-docs repo.**

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Changes can be seen here :** https://latestlitmusdoc.netlify.app/docs/getstarted

**Checklist:**
-   [ ] Fixes #<issue number>
-   [X] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

Light Screenshot
Screenshot from 2020-12-14 16-45-25
Dark Screenshot
Screenshot from 2020-12-14 16-45-21

### CODE REVIEW

File: website/docs/cassandra-pod-delete.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ sidebar_label: Cassandra Pod Delete
 
 #### Sample Rbac Manifest
 
-[embedmd]: # "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/cassandra/cassandra-pod-delete/rbac.yaml yaml"
+[embedmd]: # "https://raw.githubusercontent.com/litmuschaos/chaos-charts/v1.10.x/charts/cassandra/cassandra-pod-delete/rbac.yaml yaml"
```
Comment (uditgaurav) [inline website/docs/cassandra-pod-delete.md:62]: The versioned link should not be in master. This change can be reverted.

Comment (uditgaurav) [inline website/docs/cassandra-pod-delete.md]: Why do we need to change the version in the main docs folder.  Should it be on versioned docs only?

Comment (Jonsy13) [reply website/docs/cassandra-pod-delete.md]: It was there in litmus-docs as well. So i added here as well.

**Review by ksatchit — APPROVED** [review]

**Comment by ksatchit** [comment]
Thanks for being patient with all the requested changes @Jonsy13 !

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**